### DISCUSSION TITLE
How to reduce header size?

### DISCUSSION THREAD

**Discussion by Complexlity** [wevm/frog-archived#359] [Q&A] [answered]
I'm having same isssues as this: https://github.com/wevm/frog/discussions/331

It seems the image loads 50% the time. Fails the other 50%. What determines the generated image header size?
The number of props?. It works fine in dev mode but when deployed (on vercel), it shows same error as his.

**Update**: Was able to reduce it by:
1. Just removing some divs (I think this is what helped).
2. Removing ternary operation codes from the image and put the responsiblity in the frame. Just pass in the value so the image does no "work".
3. Changing the props to single letters (not sure 100% if this worked).
4. Changing the actual element name to single letter (also not sure 100% if this worked).
5. Remove the state from the url and into frame state instead

**Comment by dalechyn** [comment, marked as answer]
You need to use https://frog.fm/concepts/image-handler for this case.
